tests/drivers/mm/sys_mm_drv_bank/testcase.yaml

```diff
@@ -0,0 +1,4 @@
+tests:
+  drivers.mm.sys_mm_drv_bank:
+    tags: drivers mm
+    filter: CONFIG_MM_DRV
```

Comment:
`filter: CONFIG_MM_DRV` does not solve the issue.   I think adding something like:

```
filter: dt_compat_enabled("intel,adsp-tlb") or dt_compat_enabled("intel,adsp-mtl-tlb")
```

We need to add this to `tests/drivers/mm/sys_mm_drv_api/testcase.yaml`

Reply:
Gotcha. Hopefully this will do it. :)